src/components/SidebarGroup/SidebarGroupHeader/SidebarGroupHeader.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
```

Comment:
This doesn't need to be it's own component. It can just be a part of the "SidebarGroup" component.